guides/puppet-web-apps/html-views/README.md

```diff
@@ -0,0 +1,98 @@
+# HTML Views
+
+This page gives an explaination of partial HTML views and view-models, or, in other words, nested pages.
+
+## Partial HTML View
+
+A Partial HTML View is a valid HTML document that contains at least one `<template>` tag and can be loaded with the [`<imported-template>`](https://github.com/Juicy/imported-template) web component. The `<template>` tag is cached on the client and stamped out as many times as needed.
```

Comment:
While technically correct, I think that we consider [`<imported-template>`](https://github.com/Juicy/imported-template) an implementation detail from now on.

Please change this paragraph to say:

```
...can be loaded with the [`<starcounter-include>`](https://github.com/Starcounter/starcounter-include) web component
```